src/debugger/adapter/DebugAdapter.js

```diff
@@ -51,7 +51,9 @@ class PrepackDebugSession extends DebugSession {
 
   _registerMessageCallbacks() {
     this._adapterChannel.registerChannelEvent(DebugMessage.PREPACK_READY_RESPONSE, (response: DebuggerResponse) => {
-      this.sendEvent(new StoppedEvent("entry", DebuggerConstants.PREPACK_THREAD_ID));
+      if (this._clientID === DebuggerConstants.CLI_CLIENTID) {
```

Comment:
Can you add a comment here for the reason?

Reply:
It does not expect a StoppedEvent on entry of the debug adapter. The engine should be paused on entry but no event should be propagated to the UI. I will add a comment here.